src/Symfony/Bridge/Twig/Extension/WorkflowExtension.php

```diff
@@ -33,22 +33,69 @@ public function getFunctions()
             new \Twig_SimpleFunction('workflow_can', array($this, 'canTransition')),
             new \Twig_SimpleFunction('workflow_transitions', array($this, 'getEnabledTransitions')),
             new \Twig_SimpleFunction('workflow_has_marked_place', array($this, 'hasMarkedPlace')),
+            new \Twig_SimpleFunction('workflow_places', array($this, 'getMarkedPlaces')),
```

Comment:
It should be `workflow_marked_places` to avoid confusion as you stated in the previous PR.